index.js

```diff
@@ -1,102 +1,29 @@
-/* eslint max-classes-per-file: ["error", 2] */
-const booksList = document.getElementById('books-list');
+import { Book, Books } from './modules/books.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+import {
+  showForm, showContact, showList, listLink, formLink, contactLink, formSection, contact,
+} from './modules/navigation.js';
+
 const form = document.getElementById('form');
 const title = document.getElementById('title');
 const author = document.getElementById('author');
 const time = document.getElementById('time');
-const list = document.getElementById('list');
-const formSection = document.getElementById('form-section');
-const contact = document.getElementById('contact');
-const listLink = document.getElementById('list-link');
-const formLink = document.getElementById('form-link');
-const contactLink = document.getElementById('contact-link');
-
-formSection.style.display = 'none';
-contact.style.display = 'none';
 
-function showForm() {
-  formSection.style.display = 'block';
-  contact.style.display = 'none';
-  list.style.display = 'none';
-}
-
-function showContact() {
-  contact.style.display = 'block';
-  formSection.style.display = 'none';
-  list.style.display = 'none';
-}
```

Comment:
- [ ] Great job on adding the` DateTime` as requested by the previous reviewer however after the fix the `Add book` form has disappeared. Please make sure that the rest of the functionalities like displaying, adding and removing books are working after the fix.

<img width="1097" alt="Screenshot 2023-03-28 at 12 02 33 AM" src="https://user-images.githubusercontent.com/44798044/228042313-ba2fa64c-985c-489e-be9e-fd996d624e91.png">